### PR TITLE
feat(manager) : big fixes for manager revamp phase 3

### PR DESCRIPTION
**Pull request by nikhitamore**
## Explain the feature/fix

1. Settings Page toggle button
2. Web property page redirection from the sidebar and actual onclick property fixed
3. Fixed selected property name in dropdown in the sidebar 
4. Moved pagination to the top side as the view logs section was overlapping.
5. Make the view logs section aligned in the center. 

 

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by soumyadip007** [comment]
Hi @nikhitamore , I've marked this PR as draft, please make it ready for review when the rest of the changes are done.

**Comment by nikhitamore** [comment]
> Hi @nikhitamore , I've marked this PR as draft, please make it ready for review when the rest of the changes are done.

Hi @soumyadip007 
I have pushed necessary changes on the PR, kindly review .

File: packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx [hunk 1]
```diff
@@ -24,6 +25,22 @@ type SNProps = {
   isActive: boolean;
 };
 
+function getIdentifier(identifier: string) {
```
Comment (soumyadip007) [inline packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx:28]: Please convert it to `GenereateIdentifier` in the next pr

File: packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx [hunk 2]
```diff
@@ -91,22 +108,20 @@ export const SideBar = () => {
...
-                    : 'Select Property'
-                }
+                title={selectedProperty || 'Select Web Property'} // Use the selectedProperty state
                 groupId="nav-expanded-web-property"
                 isExpanded={isExpanded}
               >
                 {filteredWebProperties?.map((property) => (
                   <NavItem key={property.title}>
-                    <Link href={`/properties/${property.title}`} passHref>
+                    <Link href={`/properties/${getIdentifier(property.title)}`} passHref>
```
Comment (soumyadip007) [inline packages/manager/src/layouts/AppLayout/components/SideBar/SideBar.tsx:120]: Please make it `generateIdentifier`

**Review by soumyadip007 — APPROVED** [review]

**Pull request merged**